stograde/student/markdownify/process_file.py

```diff
@@ -139,14 +141,13 @@ def process_file(filename, *, steps, options, spec, cwd, supporting_dir, interac
     if not should_continue or not steps or options['web']:
         return results
 
-    should_continue = test_file(filename,
-                                spec=spec,
-                                results=results,
-                                options=options,
-                                cwd=cwd,
-                                supporting_dir=supporting_dir,
-                                interact=interact)
-    if not should_continue:
-        return results
+    test_file(filename,
```

Comment:
Why did you remove the return value?

Reply:
The return wasn't being used for anything once that check was removed. And the check did the same thing no matter if it evaluated to true or false